website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,129 @@
+If something is wrong with the launched node, Kubernetes will automatically migrate the pods to a new node.
+
+Once Karpenter brings up a node, that node is available for the Kubernetes scheduler to schedule pods on it as well.
+This is useful if there is additional room in the node due to imperfect packing shape or as workloads finish over time.
+
+## Cloud provider
+Karpenter makes requests to provision new nodes to the associated cloud provider.
+The first supported cloud provider is AWS, although Karpenter is designed to work other cloud providers as well.
+Separating Kubernetes and AWS-specific settings allows Karpenter a clean path to integrating with other cloud providers.
+
+While using Kubernetes well-known labels, the provisioner can set some values that are specific to the cloud provider.
+So, for example, to include a certain instance type, you could use the Kubernetes label `node.kubernetes.io/instance-type`, but set its value to an AWS instance type (such as `m5.large` or `m5.2xlarge`).
+
+Choosing spot instances is one feature that is specific to AWS.
+Cloud providers may choose to implement support for additional vendor specific labels like `node.k8s.aws/capacity-type`.
+See [AWS labels](/docs/cloud-providers/aws/) for details.
+
+## Kubernetes cluster autoscaler
+Like Karpenter, [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) is
+designed to add nodes when requests come in to run pods that cannot be met by current capacity.
+Cluster autoscaler is part of the Kubenetes project, with implementations by most major Kubernetes cloud providers.
+By taking a fresh look at provisioning, Karpenter offers the following improvements over the older cluster autoscaler:
+
+* **Designed to handle the full flexibility of the cloud**:
+Cluster autoscaler was not built with the flexibility to handle hundreds of instance types, zones, and purchase options.
+Karpenter has the ability to efficiently address the full range of instance types available through AWS.
+
+* **Adds scheduling**: Cluster autoscaler doesn’t bind pods to the nodes it creates.
+A node that Karpenter creates knows to run the bound pods immediately when the node comes up.
+I doesn't have to wait for the scheduler or for the node to become ready.
+It can start pulling the container images it needs immediately.
+This can save seconds of performance.
+
+## Constraints
+Understanding the concept of layered constraints is key to using Karpenter.
+With no constraints defined in provisioners and none requested from pods being deployed, Karpenter on AWS is free to choose from the entire universe of features available from EC2.
+Nodes can be created using any instance type and run in any zones.
+On-demand or spot instances can be used.
+
+The effects of layering constraints are applied when a pod makes requests for compute resources that cannot be met by current capacity.
```

Comment:
I'd drop this since we already cover when karpenter kicks in above (unschedulable)